railties/test/application/configuration_test.rb

```diff
@@ -3166,6 +3166,23 @@ class MyLogger < ::Logger
       assert_includes Rails.application.config.hosts, ".localhost"
     end
 
+    test "ActiveStorage.transformations_for_format is an empty hash by default" do
+      app "development"
+      assert_equal Hash.new, ActiveStorage.transformations_for_format
+    end
+
+    test "ActiveStorage.transformations_for_format can be configured via config.active_storage.transformations_for_format" do
+      app_file "config/environments/development.rb", <<-RUBY
+        Rails.application.configure do
+          config.active_storage.transformations_for_format[:jpg] = { format: "jpg" }
```

Comment:
This configuration is akin to what we'd do before this change. Can we change it to another option that wasn't possible before, so we properly assert the new behavior?

Reply:
Oh, or unless we'd default to `{ format: "png" }` for `:jpg`s? If so, disregard the comment above here.

Reply:
I'm a bit confused about this comment. From my understanding, here I'm supposed to check if whatever devs configure in their `environment` files ends in the ActiveStorage's config, is that correct? 

Since `transformations_for_format` is a new option and defaults to `{}`, I assumed that the value I used (eg: `[:jpg] = { format: "jpg" }` was not important, and that the important thing was that `ActiveStorage. transformations_for_format` had that value in the end.

Reply:
Got it, you're right. I missed that.